R/read.R

```diff
@@ -466,9 +477,12 @@ readVizgen <- function(data_dir,
                         BPPARAM = BPPARAM,
                         # use mid z section
                         MoreArgs = list(z = ifelse(z == "all", 3L, z)))
-      polys <- if (length(polys) == 1L) polys[[1]] else rbindlist(polys)
+      polys <- if (length(polys) == 1L) polys[[1]] else rbindlist(polys) |> st_as_sf()
       polys$Type <- "cell"
-      suppressWarnings(sfarrow::st_write_parquet(polys))
+      parq_file <- file.path(data_dir, "cellpose_micron_space.parquet")
```

Comment:
`'.hdf5'` file are from older version and are Watershed segmentations, naming it like `"cellpose_micron_space.parquet"` would be confusing. I will replace by "hdf5s_micron_space.parquet" or similar